add-on/panels/js/panel.jsx

```diff
@@ -755,10 +737,6 @@ loop.panel = (function(_, mozL10n) {
         return this._renderLoadingRoomsView();
       }
 
-      if (!this.state.rooms.length) {
-        return this._renderNoRoomsView();
```

Comment:
This logic was short circuiting the below. Do you have a screenshot of what no rooms looks like now? I would guess it would show the Recently Browsed header text with this removal.